import-service/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "import-service",
+  "version": "1.0.0",
+  "description": "Serverless webpack example using Typescript",
+  "main": "handler.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "dependencies": {
+    "aws-sdk": "^2.792.0",
```

Comment:
It will be better to move this dependency to the `devDependencies` (or configure serverless configuration to exclude this module) to reduce bundle size.
All deployed AWS lambda functions have access to this package by default.

Reply:
Thank you for your note!